packages/date/src/index.js

```diff
@@ -5,6 +5,12 @@ import momentLib from 'moment';
 import 'moment-timezone/moment-timezone';
 import 'moment-timezone/moment-timezone-utils';
 
+/**
+ * Object containing `moment` instance.
+ *
+ * @typedef {Object} Moment
```

Comment:
@dsifford - does it make sense? I have no idea how those types should be defined, to be honest :)

Reply:
Moment [provides bundled type definitions](https://github.com/moment/moment/blob/develop/moment.d.ts), so you should just be able to import it the way you do with `@wordpress` packages.

Reply:
Thanks, it's fixed now 👍